website/docs/r/iothub.html.markdown

```diff
@@ -294,6 +296,14 @@ A `fallback_route` block supports the following:
 
 A `file_upload` block supports the following:
 
+* `authentication_type` - (Optional) Type used to authenticate against the storage account. Possible values are `keyBased` and `identityBased`. Defaults to `keyBased`.
+
+* `identity_id` - (Optional) ID of the User Managed Identity used to authenticate against the storage account.
+
+-> **NOTE:** `identity_id` can only be specified when `authentication_type` is `identityBased`. It must be one of the `identity_ids` of the Iot Hub. If not specified when `authentication_type` is `identityBased`, System Assigned Managed Identity of the Iot Hub will be used.
```

Comment:
```suggestion
-> **NOTE:** `identity_id` can only be specified when `authentication_type` is `identityBased`. It must be one of the `identity_ids` of the Iot Hub. If `identity_id`is omitted when `authentication_type` is `identityBased`, then the System Assigned Managed Identity of the Iot Hub will be used.
```
Furthermore does `azurerm_iothub` have the plural property `identity_ids`?

Reply:
`identity_ids` is introduced by #14354 in `"identity": commonschema.SystemAssignedUserAssignedIdentity()`